src/pages/Swap/index.tsx

```diff
@@ -69,10 +69,13 @@ import { useIsDarkMode } from '../../theme/components/ThemeToggle'
 import { OutputTaxTooltipBody } from './TaxTooltipBody'
 import { UniswapXOptIn } from './UniswapXOptIn'
 
-export const ArrowContainer = styled.div`
+export const ArrowContainer = styled.button`
```

Comment:
Rather than change this to a button, which requires additional styles, would it be feasible to render the actual component using `as="button"` ([docs](https://styled-components.com/docs/api#as-polymorphic-prop))?